test/models/post_test.rb

```diff
@@ -1,7 +1,18 @@
 require 'test_helper'
 
 class PostTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'presence of post title and description' do
+    @post = FactoryGirl.create(:post)
+    assert_equal @post.valid?, true, 'valid post'
```

Comment:
@anijat-cuelogic : The message should contain the text which we need to display **if the test case fails**. Refer [this] (http://ruby-doc.org/stdlib-2.1.0/libdoc/test/unit/rdoc/Test/Unit/Assertions.html#method-i-assert_equal)